src/TGameDetails.h

```diff
@@ -444,6 +444,29 @@ qsl("<a href='https://abandonedrealms.com'>Website</a><br>"
                  "Busted. You can also use it as a starting point to create automated tests for your "
                  "own profiles!")},
 
+            {qsl("Accursed Lands"),
+             qsl("mud.accursed-lands.com"),
+             7979,
+             true,
+             qsl("<a href='https://www.accursed-lands.net'>https://www.accursed-lands.net</a><br>"
```

Comment:
I don't think we need to show the scheme in the (visible) anchor text:
```suggestion
             qsl("<a href='https://www.accursed-lands.net'>www.accursed-lands.net</a><br>"
```